deploy/helm/pulumi-operator/values.yaml

```diff
@@ -17,36 +17,28 @@ image:
   # -- The image tag to pull, default: `Chart.appVersion`
   tag: ""
 
+# -- The image pull secrets
+imagePullSecrets: ""
+
 controller:
-  # -- Graceful shutdown timeout duration, default: `5m`
-  gracefulShutdownTimeoutDuration: 5m
-  # -- Max concurrent reconciles, default: `10`
-  maxConcurrentReconciles: "10"
-  # -- Pulumi infer namespace, default: `1`
-  pulumiInferNamespace: "1"
-  # -- Kubernetes Cluster Domain, default: `cluster.local`
-  kubernetesClusterDomain: cluster.local
-
-  # -- List of arguments to pass to the operator
-  args:
-    - --zap-level=error
-    - --zap-time-encoding=iso8601
+  # -- Log Level ('debug', 'info', 'error', or any integer value > 0 which corresponds to custom debug levels of increasing verbosity)
+  logLevel: error
 
 # -- Extra Environments to be passed to the operator
 extraEnv: []
 
-# -- Create a ClusterRole resource for the node-red pod. default: false
-createClusterRole: false
-
-## -- Enable this and set the rules: to whatever custom rules you want for the Cluster Role resource.
-clusterRoleRules:
-  # -- Enable custom rules for the application controller's ClusterRole resource default: false
-  enabled: false
-  # -- List of custom rules for the application controller's ClusterRole resource default: []
-  rules: []
-
-# -- The image pull secrets
-imagePullSecrets: ""
+# RBAC settings
+rbac:
+  # -- Specifies whether RBAC resources should be created
+  create: true
+  # -- Specifies whether cluster roles and bindings should be created
+  createClusterRole: true
+  # -- Specifies whether namespaced roles and bindings should be created
+  createRole: false
```

Comment:
one either creates a clusterrole or a role for the controller manager, depending on whether you want a namepaced installation of the operator.  There's still some follow-up work needed, but the helm piece is in place.